src/MOI_wrapper.jl

```diff
@@ -242,7 +242,7 @@ function load_constraint(
         index::MOI.ConstraintIndex, model::CbcModelFormat,
         mapping::MOIU.IndexMap, func::MOI.VectorOfVariables,
         set::MOI.SOS1{Float64})
-    var_indices = [v.value for v in func.variables]
+    var_indices = [Int32(v.value) for v in func.variables]
```

Comment:
Why not `Int32[` ? That way Julia doesn't even have to try to infer the return type and it also converts

Reply:
changed